content/hi/_index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
+abstract: "स्वचालित कंटेनर डिप्लॉयमेंट, स्केलिंग और प्रबंधन"
+cid: home
+sitemap:
+  priority: 1.0
+---
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+[कुबेरनेट्स]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}), K8s के रूप में भी जाना जाता है, कंटेनरीकृत एप्लीकेशन के डिप्लॉयमेंट, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन-सोर्स सिस्टम है।
+
+यह आसान प्रबंधन और खोज के लिए तार्किक इकाइयों में एक एप्लीकेशन बनाने वाले कंटेनरों को समूहित करता है। कुबेरनेट्स [Google पर उत्पादन कार्यभार चलाने के १५ वर्षों के अनुभव](http://queue.acm.org/detail.cfm?id=2898444), पर निर्माण करता है, जो समुदाय के सर्वोत्तम-नस्लीय विचारों और प्रथाओं के साथ संयुक्त है।
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="scalable" %}}
+#### ग्रह पैमाने <!--Planet Scale-->
+
+उन्हीं सिद्धांतों पर डिज़ाइन किया गया है जो Google को एक सप्ताह में अरबों कंटेनर चलाने की अनुमति देता है, कुबेरनेट्स आपकी ऑप्स टीम को बढ़ाए बिना स्केल कर सकता है।
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="blocks" %}}
+#### कभी नहीं बढ़ना <!--Never Outgrow-->
+
+चाहे स्थानीय स्तर पर परीक्षण हो या वैश्विक उद्यम चलाना, कुबेरनेट्स आपके एप्लीकेशन को लगातार और आसानी से वितरित करने के लिए आपके साथ बढ़ता है, चाहे आपकी आवश्यकता कितनी भी जटिल क्यों न हो।
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature image="suitcase" %}}
+#### K8s कहीं भी चलाएँ
+
+कुबेरनेट्स ओपन सोर्स है जो आपको ऑन-प्रिमाइसेस, हाइब्रिड या पब्लिक क्लाउड इन्फ्रास्ट्रक्चर का लाभ उठाने की स्वतंत्रता देता है, जिससे आप आसानी से वर्कलोड को उस स्थान पर ले जा सकते हैं जहां यह आपके लिए मायने रखता है।
+
+{{% /blocks/feature %}}
+
+{{< /blocks/section >}}
+
+{{< blocks/section id="video" background-image="kub_video_banner_homepage" >}}
+<div class="light-text">
+<h2>150+ माइक्रोसर्विसेज को कुबेरनेट्स में माइग्रेट करने की चुनौतियाँ</h2>
+        <p>By Sarah Wells, Technical Director for Operations and Reliability, Financial Times</p>
```

Comment:
```suggestion
        <p>Sarah Wells द्वारा, Technical Director for Operations and Reliability, Financial Times</p>
```

Reply:
how does this look @sftim ?
I am not able to think of a way to translate `for` separately, without changing the whole sentence and plus its not making much sense!! Some ideas & suggestions would help here 🙂